Cargo.toml

```diff
@@ -207,3 +207,8 @@ pallet-beefy-mmr = { git = "https://github.com/composableFi/substrate", branch =
 beefy-merkle-tree = { git = "https://github.com/composableFi/substrate", branch = "polkadot-v0.9.16" }
 pallet-nicks = { git = "https://github.com/composableFi/substrate", branch = "polkadot-v0.9.16" }
 pallet-staking-reward-fn = { git = "https://github.com/composableFi/substrate", branch = "polkadot-v0.9.16" }
+
+[workspace.metadata.dylint]
+libraries = [
+    { git = "https://github.com/kaiserkarel/substrace" },
```

Comment:
ok. so would prefer this to be part of composable repo for easy maintenance until tool will become stable amid several upgrade of substrate.

Reply:
Yeah I can fork the repo to composable, do dev on my branch then sync regularly for example, or just migrate it over entirely.